src/openai/cli/_api/audio.py

```diff
@@ -65,6 +65,9 @@ def transcribe(args: CLITranscribeArgs) -> None:
         with open(args.file, "rb") as file_reader:
             buffer_reader = BufferReader(file_reader.read(), desc="Upload progress")
 
+        # add the file extension otherwise file format detection fails
+        buffer_reader.name = "audio." + args.file.split(".")[-1]
+
         model = get_client().audio.transcriptions.create(
             file=buffer_reader,
```

Comment:
nit: I think this is a cleaner solution and I think it should still work but I haven't tested it

```suggestion
        model = get_client().audio.transcriptions.create(
            file=(args.file, buffer_reader),
```

Reply:
I just tested it and indeed your proposition works perfectly. I'll push an updated fix right away then. Thanks!